src/CaptureFileInfo/CaptureFileInfoTest.cpp

```diff
@@ -85,4 +85,23 @@ TEST(CaptureFileInfo, Touch) {
   EXPECT_GE(capture_file_info.LastUsed(), now);
 }
 
+TEST(CaptureFileInfo, FileSize) {
+  {
+    const QString non_existing_path{"test/path/file.ext"};
+
+    CaptureFileInfo capture_file_info{non_existing_path};
+
+    EXPECT_EQ(capture_file_info.FileSize(), 0);
+  }
+  {
+    const std::filesystem::path path =
+        orbit_base::GetExecutableDir() / "testdata" / "not_existing_test_file.txt";
```

Comment:
`not_existing_test_file`? Does that file actually exist?

Reply:
I rebased wrong. This was a mistake that I fixed, but the fix ended up one commit later. This means also the test will fail at this commit. I will fix that

Reply:
Feel free to force push.